templates/cvAdvisor.html.twig

```diff
@@ -23,32 +23,54 @@
                     </div>
                     <div class="blocInfo" id="competences">
                         <h2>Compétences clés</h2>
-                        <p>Marketing / Communication & réseaux sociaux, Stratégie / Business Developpement / Vision stratégique</p>
+                        <div class="tags">
+                            {% for tag in advisor.tags %}
+                                {% if tag.category.id == 2 %}
```

Comment:
Rather use a name than an id (imagine you have to change the database)

Reply:
Fixed